feature/gnoi/system/tests/copying_debug_files_test/copying_debug_files_test.go

```diff
@@ -86,9 +91,26 @@ func TestCopyingDebugFiles(t *testing.T) {
 	t.Logf("Wait 60 seconds for process to restart ...")
 	time.Sleep(60 * time.Second)
 
-	componentName := map[string]string{"name": components[dut.Vendor()]}
+	if deviations.ChassisGetRPCUnsupported(dut) {
+		var activeRp string
+		rpList := comps.FindComponentsByType(t, dut, oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_CONTROLLER_CARD)
+		t.Logf("Found RP list: %v", rpList)
+		if len(rpList) < 2 {
+			activeRp = "0/RP0/CPU0"
```

Comment:
add switch statement for dut.Vendor like:
```
		switch dut.Vendor() {
		case ondatra.CISCO:
```

Reply:
could you explain why the switch is needed? what would the default case be here?

the code under deviation is applicable only to Cisco at the moment. it is implied that active controller_card would be 0/RP0/CPU0 for Cisco via this block.

Reply:
It's needed because your implementation of this deviation is cisco specific.  ie: that particular deviation contains ios-xr, cisco 8800 specific strings in it.    The default case is to do nothing.  Other vendors could assert the deviation and add  cases as needed.